packages/rich-list-api-client/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": false,
+  "name": "@defichain/rich-list-api-client",
+  "version": "0.0.0",
+  "description": "DeFiChain Jellyfish Ecosystem",
+  "repository": "DeFiCh/jellyfish",
+  "bugs": "https://github.com/DeFiCh/jellyfish/issues",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b ./tsconfig.build.json"
+  },
+  "dependencies": {
+    "@defichain/ocean-api-client": "0.0.0"
+  },
+  "peerDependencies": {
+    "defichain": "^0.0.0"
+  }
+}
```

Comment:
Best to separate everything in `rich-list-api-client` as a barebone PR, easier to get them merged in iteratively.

Reply:
Wanted to use the response interface from `api-client`. But literally only one, copied over and trim the package.